SwiftCarousel.swift

```diff
@@ -353,20 +346,14 @@ public class SwiftCarousel: UIView {
     }
     
     private func selectItem(choice: Int, animated: Bool, force: Bool) {
+        var index = choice
         if !force {
-            guard choice < choices.count / 3 else { return }
-        }
-        
-        var min: Int = originalChoicesNumber
-        var index: Int = choice
-        
-        if let realSelectedIndex = self.realSelectedIndex where !force {
-            for choiceIndex in choice.stride(to: choices.count, by: originalChoicesNumber) {
-                if abs(realSelectedIndex - choiceIndex) < min {
-                    index = choiceIndex
-                    min = abs(realSelectedIndex - choiceIndex)
-                }
+            // allow scroll only in the range of original items
+            guard choice < choices.count / 3 else {
```

Comment:
just simplified the math, not sure why the stride and abs(), maybe now it is not correct, but basically I just want to add one 'segment' so the index will be in the 'middle segment'